src/qtmozembed/declarativewebpage.cpp

```diff
@@ -95,6 +95,12 @@ DeclarativeWebPage::DeclarativeWebPage(QObject *parent)
     connect(&m_grabWritter, SIGNAL(finished()), this, SLOT(grabWritten()));
     connect(this, SIGNAL(urlChanged()), this, SLOT(onUrlChanged()));
     connect(this, &QOpenGLWebPage::contentHeightChanged, this, &DeclarativeWebPage::updateViewMargins);
+    connect(this, &QOpenGLWebPage::loadedChanged, this, [=]() {
```

Comment:
Good point. Dropped 3rd argument (2nd this) and used ```this``` instead of ```=```.  In real life kind of simple lambda case there would not be any diff as we're not copying anything inside the lambda.